kong/init.lua

```diff
@@ -440,6 +748,23 @@ function Kong.log()
   core.log.after(ctx)
 end
 
+function Kong.mock_log()
+end
+
+function Kong.mock_service()
+  local request = {
+    headers = ngx.req.get_headers(nil, true),
+  }
+
+  local args   = require "kong.api.arguments".load({ decode = false })
```

Comment:
I think the `require` should be moved to the top of the file, in order to make dependencies explicit.